__tests__/routes/social.test.js

```diff
@@ -0,0 +1,110 @@
+import request from 'supertest';
+import nock from 'nock';
+import dotenv from 'dotenv';
+import { facebookUser, googleUser, twitterUser } from '../mocks/db.json';
+import { urlPrefix } from '../mocks/variables.json';
+import app from '../../app';
+
+dotenv.config();
+jest.setTimeout(30000);
+
+describe('SOCIAL AUTHENTICATION', () => {
+  describe('User login through Twitter', () => {
+    beforeAll(async () => {
+      await nock('https://www.twitter.com/')
+        .filteringPath(() => '/')
+        // .persist()
```

Comment:
Why is this line commented?

Reply:
Ohhhh I was testing to see it will make any difference if I omit it or not, let me remove it.